[16.0][OU-ADD]  website_hr_recruitment: Nothing to do

nguyenvietlam0640 opened the pull request.

nguyenvietlam0640 committed:
[MIG] website_hr_recruitment: migrate 15 to 16

legalsylvain commented:
/ocabot migration website_hr_recruitment

<dependency>Depends on : 

- hr_recruitment : #4296</dependency>

pedrobaeza committed:
[OU-ADD] website_hr_recruitment: Migration scripts

The column hr.job>job_details requires to be pre-created for not
having it with wrong defaults for existing jobs.

TT45243

pedrobaeza commented:
I have pushed an extra commit doing the needed pre-creation of the `job_details` column for avoiding a non-desired default value that will be shown on existing jobs.
